backend/de.metas.manufacturing/src/main/java/org/eevolution/process/RollupBillOfMaterial.java

```diff
@@ -122,19 +122,21 @@ protected String doIt()
 				.repository(bomCostCalculatorRepo)
 				.build();
 
-		final int maxLowLevel = getMaxLowLevel();
+		//@TODO the process is not used in WEB UI, need to be updated in necessary
+/*		final int maxLowLevel = getMaxLowLevel();
```

Comment:
pls throw an exception that is not supported and also pls document this on the issue page so the testing team will be aware of this too.

NOTE: this is affecting the standard costing calculation, which we no longer use.

Reply:
done